recipe/meta.yaml

```diff
@@ -114,16 +115,18 @@ outputs:
         - {{ pin_subpackage("nvmath-python", exact=True) }}
         - cupy  # TODO: remove me in the next release
         - numba 0.59.*
+        - pynvjitlink 0.1.14.*
         - cuda-python
         - cuda-nvcc
         - cuda-nvrtc
         - cuda-cudart-dev
         - cuda-cccl
-        - mathdx 24.04.0.*
+        - mathdx 24.04.0.* *_1
         - cuda-version >=12.0.*,<12.4.a0
     test:
       imports:
         - nvmath
+        - nvmath.device
```

Comment:
If all packages are done right, this import should pass (which is what's being tested in this PR).